core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/TelemetryConverter.java

```diff
@@ -13,10 +17,15 @@
 import com.sequenceiq.common.api.telemetry.response.LoggingResponse;
 import com.sequenceiq.common.api.telemetry.response.TelemetryResponse;
 import com.sequenceiq.common.api.telemetry.response.WorkloadAnalyticsResponse;
+import com.sequenceiq.sdx.api.model.SdxClusterResponse;
 
 @Component
 public class TelemetryConverter {
 
+    private static final String DATABUS_HEADER_SDX_ID = "databus.header.sdx.id";
```

Comment:
should we use configuration properties here? will this change?

Reply:
attributes are overriding the cm configs (dynamic way) for TP, this configs should go into safety valve unfortunately so no specific configs for these in CM. therefore if it will change on CM side, we will need to change that, although as i have checked, these are databus header names also dynamic in CM, but not changed for long time